plasmapy/physics/parameters.py

```diff
@@ -595,11 +564,19 @@ def collision_rate_electron_ion(T_e,
         String signifying a particle type of the field ions, including charge
         state information.
 
+    V : ~astropy.units.Quantity, optional
+        The relative velocity between particles.  If not provided,
+        thermal velocity is assumed: :math:`\mu V^2 \sim 2 k_B T`
+        where `mu` is the reduced mass.
+
     coulomb_log : float or dimensionless ~astropy.units.Quantity, optional
         Option to specify a Coulomb logarithm of the electrons on the ions.
         If not specified, the Coulomb log will is calculated using the
         `~plasmapy.physics.transport.Coulomb_logarithm` function.
 
+    coulomb_log_method : string, optional
+        Method used for Coulomb logarithm calculation (see that function
```

Comment:
Can you link to the function from here?

Reply:
I've been reading a bit on the Numpydoc standard. Apparently it's supposed to be readable and concise first, and pretty (with Sphinx, links) etc. second.

Whoops, I guess.

Apparently Numpy has a whole bunch of preprocessors set up on their docstrings to prettify them
before sending them to Sphinx to get rendered. It's an interesting technique we might have to look into some time. For now, let's keep things this way.

Anyway, to the point... it's linked literally two lines above. I think people will manage.

I did fix the whitespace in here a bit, though, so thanks!